src/cache.rs

```diff
@@ -350,7 +352,8 @@ impl Cache {
 
         if *state > EntryState::Typechecked {
             Ok(CacheOp::Cached(()))
-        } else if *state == EntryState::Parsed {
+        } else if [EntryState::Parsed,EntryState::ImportsResolved]
+            .contains(state) {
             type_check(t, global_env, self)?;
```

Comment:
I think it will have the same behavior as before, so no errors only it will not typecheck the imports. I was thinking may be someone want to typecheck a file without resolving the imports. I don't know. But clearly in an execution context, it make no sens.

Reply:
also, I don't do the imports resolution for the stdlib... my bad. The weqr thing is that the test do not fail

Reply:
I would lend to always do import resolution before typechecking. If at some point we need to have one without the other, we can revisit, but in the meantime at least we are not at risk of forgetting import resolution by accident. That's not a dramatic issue though, so do as you think is best.